spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala

```diff
@@ -306,7 +306,7 @@ object SGD {
     override def updateHyperParameter[T](optimMethod: SGD[T]): Unit = {
       val nevals = optimMethod.state.get[Int]("evalCounter").getOrElse(0)
       val lr = optimMethod.learningRate
-      val polyIter = nevals - excludeIterations
+      val polyIter = nevals // fix: should have no exclude iterations.
```

Comment:
this is no longer consistent with line#295

Reply:
@i8run  This change lead the learningRate of TrainInceptionV1 inconsistent, if warmupEpoch and maxLR is set.